src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Processing your login: {}", login);
```

Comment:
you should use `logger.debug` here, as I mentioned earlier

in addition -  `your login` - bad statement, since our app doesn't know anything about him/her/etc
you should better say `Processing login`

Reply:
Oooh right. This is what youve meant. Misunderstood you the first time :)